src/App.tsx

```diff
@@ -14,7 +48,19 @@ export const App = () => {
       </div>
 
       <div className="sidebar">
-        <FindMovie />
+        <FindMovie
+          setQuery={setQuery}
+          movieFound={someMovie}
+          setMovies={setMovies}
+          setMovie={setMovie}
+          query={query}
```

Comment:
Pass the 'movies' state to 'FindMovie' instead of the 'setMovies' function to handle duplicate prevention within 'App' component.